services/venice-server/src/test/java/com/linkedin/venice/listener/StorageReadRequestHandlerTest.java

```diff
@@ -2,6 +2,7 @@
 
 import static com.linkedin.venice.read.RequestType.SINGLE_GET;
 import static com.linkedin.venice.router.api.VenicePathParser.TYPE_STORAGE;
+import static io.netty.handler.codec.http.HttpResponseStatus.*;
```

Comment:
Let's remove static import.

Reply:
Sure, hate Intellij not saving import rules :(